src/logical/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (number1,number2, operator) => {
+    if (number2 === undefined) return number1;
+    if (number1 === undefined) return number2;
+
+    Big.DP = 10;
+    const a = Big(number1);
+    const b = Big(number2);
+
+    try{
```

Comment:
**[OPTIONAL]**
- ```try and catch``` with ```switch```, isn't it an overkill? why not let the default method capture the error🤔 .